src/components/Modals/Metadata/Metadata.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { Box, DataTable, Image, Text } from 'grommet'
+import { Close } from 'grommet-icons'
+import styled from 'styled-components'
+import { PlainButton } from '@zooniverse/react-components'
+import data from './mockData'
+import Satellite from 'images/satellite_map.png'
+import ConditionalLink from './components/ConditionalLink'
+
+const StyledDataTable = styled(DataTable)`
+  width: 100%;
+
+  thead { display: none; }
+  th { text-align: right; }
+`
+
+const Uppercase = styled(Text)`
+  text-transform: uppercase;
+`
+
+const columns = [{
+  property: 'key',
+  render: datum => <Uppercase color='kelp' weight='bold'>{datum.key}</Uppercase>
+}, {
+  property: 'value',
+  render: datum => <ConditionalLink color='kelp' text={datum.value} />
+}]
+
+export default function Metadata() {
+  return (
+    <Box
+      border={{ color: 'kelp' }}
+      elevation='small'
+      gap='xsmall'
+      height='32rem'
+      pad={{ horizontal: 'medium', vertical: 'small' }}
+      width='27rem'
+    >
+      <Box direction='row' justify='between'>
+        <Text color='kelp'>Subject Metadata</Text>
+        <PlainButton
+          icon={<Close color='black' size='small' />}
+          label={<Uppercase color='kelp' size='small'>Close</Uppercase>}
```

Comment:
```suggestion
          text='close'
```

Also, the `PlainButton` applies the letter spacing styling. Then, I'd suggest we make a minor update to `PlainButton`:  https://github.com/zooniverse/front-end-monorepo/blob/661bc16f39297d1eb5fc4bf5555515d973c0ac68/packages/lib-react-components/src/PlainButton/PlainButton.js#L24-L27

I made the mistake of hard coding the colors it uses, so this can change to be a prop and what it is set to as the default.

Reply:
@srallen It sounds like there are a few changes that could be made on the `PlainButton` component. Maybe the following:

- Expose the `uppercase` prop for the label component
- Allow setting a `color` prop

I can open an issue for this and make the necessary changes if that sounds like a good plan.

Reply:
Sure. Though I'm not understanding why `uppercase` is necessary to expose. The design is to use uppercase, which it is already applying.

Reply:
I made a comment about that above but now I'm seeing that `SpacedText` sets the `uppercase` prop to true by default, so I'm not sure why it isn't working for me. I'll investigate a bit.

Reply:
Ahh, using `label` overwrites the prop in `PlainButton`, I needed to use `text` instead